build_tools/ci/cpu_comparison/matmul_template/matmul_generator.py

```diff
@@ -13,12 +13,12 @@ def generate_matmul_test(output_fn, input_fn, m, n, k, lhs_rhs_type, acc_type, b
     replace["N"] = n
     replace["K"] = k
     replace["TYPE1"] = lhs_rhs_type
-    replace["TYPE2"] = acc_type
+    replace["TYPE2"] = output_type
```

Comment:
Here and elsewhere, I'd choose to keep the distinction as `acc_type` because `output_type` (as the name suggests) doesn't necessarily has to be different than `lhs_rhs_type` (eg: bf16 -> bf16 Matmul + Truncf case) - in which we get to have a higher bit-width accumulator type but the output type is same as LHS/RHS.